Classes/Controller/BackendServiceController.php

```diff
@@ -532,11 +532,13 @@ public function flowQueryAction(array $chain): string
         $createContext = array_shift($chain);
         $finisher = array_pop($chain);
 
+        $nodeContextPaths = array_unique(array_column($createContext['payload'], '$node'));
```

Comment:
i dont think we need the `array_unique` here as flowquery does permit passing twice the same node, even if stupid. And on the calling site we already use a `Set`.

So in the end i guess we dont need this whole `array_column` change at all?